docs/running_server.md

```diff
@@ -0,0 +1,16 @@
+
+# Running the server
+
+For the server to start, it must connect to a mongodb database. 
+The server reads information about this database from environment
+variables. These can be set manually, or put in a dotenv (.env) file.
+
+* `DB_URI` should be of the form `mongodb://[username:password@]host[:port]`. \
+  Note that the default port for mongodb is 27017. For testing this will likely work:
+  `mongodb://localhost:27017`
```

Comment:
It can also be a `srv` variant, we probably want to link to: https://docs.mongodb.com/manual/reference/connection-string/

Reply:
fixed in latest commit